build-wheelhouse/action.yml

```
@@ -123,6 +123,14 @@ inputs:
     default: false
     type: boolean
 
+  use-uv:
+    description: |
+      Whether to use uv for installation operations instead of pip. By default it is set
+      to ``true``.
```

Just to be consistent with the other descriptions.
```suggestion
      Whether to use uv as the default package manager instead of pip. Default value is ``true``.
```

---

Thanks for catching that @MaxJPRey